### DISCUSSION TITLE
GitHub misidentifies the technologies used

### DISCUSSION THREAD

**Discussion by Similization** [community/community#69011] [Repositories]
Bug

### Body

I wrote my code without using any html, but it seems like github misidentifies technologies/languages in my [project](https://github.com/Similization/kts-project)

**Comment by spenserblack** [comment]
You have a large HTML file [here](https://github.com/Similization/kts-project/blob/94a6dbf4e67e91468b14f09bc077665c741060f1/etc/index.html).
